.builds/linux.yml

```diff
@@ -18,13 +19,23 @@ environment:
 tasks:
   - rustup: |
       curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal
-  - test: |
-      cd alacritty
-      cargo test
   - rustfmt: |
       cd alacritty
       rustup toolchain install nightly -c rustfmt
       cargo +nightly fmt -- --check
+  - man-pages: |
+      cd alacritty
+      printf "Compiling alacritty.1.scd" > /dev/null
```

Comment:
Shouldn't this be written to stdout, instead of to /dev/null? I'd assume it's because `scdoc` doesn't reference the input file and thus errors cannot clearly identify a file.

Reply:
I'd suggest to look into the CI output, it's better to understand like that. Basically all redirection and the way output appear is not consistent, so it's just to help identify the command being run.

Reply:
that's how it looks without redirection https://builds.sr.ht/~kchibisov/job/1054018#task-man-pages

Reply:
So what's the point of having it in the first place?

Reply:
yeah, so I can at least know which command failed.